doc/user_documentation/powerful_builders.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+.. _powerful_builders:
+
+Default builders in Copr are too slow!
+======================================
+
+The default Fedora Copr workers are typically fast enough for the vast majority
+of package builds.  However, some package builds are extremely
+resource-intensive, and our default workers struggle with them.  A notable
+example is Blink_-based browsers, which can take even more than 24 hours to
+build.
+
+From a build system perspective, this isn't a problem.  You can simply specify
+``--timeout 180000`` (in seconds) and be prepared to wait for the build to
+finish.  But if you want to expedite your builds, you do have the option to
+request "powerful" builders.
+
+Pros & Cons
+-----------
+
+The "powerful" builders will handle your builds much faster (according to the
+configuration as of October 2023, they are approximately 8 times faster than the
+default builders).  The downside is that they are in limited supply.  We can
+only start a few of them, and if multiple users request them and compete with
+each other, they may spend more time in the queue, resulting in longer wait
+times.  Additionally, because these machines are costly, we don't pre-allocate
+them (as we do with the normal builders).  We only start these powerful ones
+when "enabled builds" request them (which may itself take several minutes).
+
+As a consequence, when you configure your project, chroot or package to use
+powerful builders, there's no "magical fallback to normal builders" implemented.
```

Comment:
powerful builders or high-performance builders?
We should unify the naming. Everywhere. I am +1 for high-performance

Reply:
done